pycbc/inference/models/relbin.py

```diff
@@ -169,10 +169,14 @@ def __init__(
             variable_params, data, low_frequency_cutoff, **kwargs
         )
 
+        # FIXME: This should be set properly in some way
```

Comment:
Replace with check on which interface provides the approximant

Reply:
Changed all "is_lisa" flags to check the approximant name in the list fd_det_sequence instead